models/RoutesParser.cfc

```diff
@@ -592,6 +612,27 @@ component accessors="true" threadsafe singleton {
 		}
 	}
 
+	/**
+	* return true if the metadata represents a function marked "noCbSwagger"
+	*/
+	private boolean function isAnnotatedWithTruthyNoCbSwaggerAttribute( required struct functionMetadata ) {
```

Comment:
Let's rename this method to `isCbSwaggerEnabled`

Reply:
Actually since this is within the scope of the parser, let's just call this `isDocumentationEnabled`.  Then we could hook in to other exclusions, as necessary, down the road.

Reply:
addressed